assignments/callbacks.js

```diff
@@ -27,19 +27,32 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  cb(arr.length);
 }
+getLength(items, (length) => {
+  alert(length);
+});
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  cb(arr[5])
 }
+last(items, (lastItem) => {
+  alert(lastItem);
+});
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return x + y
 }
+sumNums(5,10)
+
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return x*y
 }
+multiplyNums(5,10)
 
 function contains(item, list, cb) {
```

Comment:
function contains(item, list, cb) {
  // contains checks if an item is present inside of the given array/list.
  // Pass true to the callback if it is, otherwise pass false.
  for (let i = 0; i < list.length; i++) {
    if (list[i] === item) {
      return cb(true);
    }
  }
  return cb(false);
}

contains('Bats', items, function(flag) {
  console.log(flag); // should be false
});

contains('Notebook', items, function(flag) {
  console.log(flag); // should be true
});